InclusionBot.md

```diff
@@ -73,6 +73,8 @@ approves your changes, they will be merged and become active in the bot.
 | master/slave, master / slave                     | primary/secondary<br>primary/replica<br>leader/worker<br>main/replica                         | This term is rooted in the enslavement of human beings. We want to avoid slavery references.<br><br>See [GitHub to replace "master" with alternative term to avoid slavery references](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/) for more.                                                                                                            |
 | open the kimono, open kimono                     | share information                                                                             | This term arises from the sexualization of Asian women. We want to avoid sexist, racist, and sexual references.<br><br>See [The Problematic Origins of Common Business Jargon](https://www.fastcompany.com/90239290/six-business-phrases-that-have-racist-origins) for more.                                                                                                                                           |
 | grandfathered                                    | legacy<br>included retroactively                                                              | This term arose from voter restrictions that prevented Black people from voting. We want to avoid slavery and racist oppression references.<br><br>See [The Problematic Origins of Common Business Jargon](https://www.fastcompany.com/90239290/six-business-phrases-that-have-racist-origins) for more.                                                                                                               |
+| Goddamn, goddam, goddamnit, goddammit            | awful, terrible, wretched                                                                     | This term implies that God condemns, curses or reproaches. We want to show respect towards all religious traditions.                                                                                                                                                                                                                                                                                                   |
```

Comment:
this isn't the updated language that @mgwalker suggested, is it? shouldn't it be
`
This term can be offensive or triggering to some religious traditions because it implies a deity cursing or reproaching people. We want to show respect towards all religious traditions
`

?

Reply:
Yes, I updated the [main document](https://docs.google.com/document/d/1iQT7Gy0iQa7sopBP0vB3CZ56GhyYrDNUzLdoWOowSHs/edit?usp=sharing), it seems I forgot to update it here. Thank you for the catch. I think I'll need to recreate the PR, as my first commit was not verified.